hail/src/main/scala/is/hail/methods/VEP.scala

```diff
@@ -149,18 +150,12 @@ object VEP {
                           warn(s"No CSQ INFO field for VEP output variant ${ VariantMethods.locusAllelesToString(vepLocus, vepAlleles) }.\nVEP output: $s.")
                           null
                       }
-                      (Annotation(locus, alleles), a)
+                      Some((Annotation(locus, alleles), a))
                     case None =>
                       fatal(s"VEP output variant ${ VariantMethods.locusAllelesToString(vepLocus, vepAlleles) } not found in original variants.\nVEP output: $s")
                   }
                 } else {
-                  val jv = try {
-                    JsonMethods.parse(s)
-                  } catch {
-                    case _: JsonParseException =>
-                      log.warn(s"vep: failed to parse json: $s")
-                      null
-                  }
+                  Try(JsonMethods.parse(s)).map {jv =>
```

Comment:
So, I guess I'm fine with this here because it isn't performance sensitive, but Try is up there with Option since it allocates a Try object to wrap the result and/or error, while the code looks exactly like a try ... catch.